docs/performance-metrics.md

```diff
@@ -0,0 +1,98 @@
+# OpenShift ETCD Performance Metrics
+
+We often get complaints about ETCD performance being bad, this document should help track down the culprit and mitigate the issue at hand using Prometheus Metrics.
+
+## Common Bottlenecks
+
+### Memory
+
+ETCD uses bbolt, which stores everything as a single memory mapped file. In OpenShift this is up to 8 GiB big, this file is organized with a B+Tree and chunked into discrete 4k pages which can be flagged in-use or not. On top of that, several in-memory caches/indices on both ETCD and API server are being used to facilitate fast multi-version access and label/field selection respectively.
```

Comment:
```suggestion
etcd uses bbolt, which stores everything as a single memory mapped file. In OpenShift this is up to 8 GiB big, this file is organized with a B+Tree and chunked into discrete 4k pages which can be flagged in-use or not. On top of that, several in-memory caches/indices on both ETCD and API server are being used to facilitate fast multi-version access and label/field selection respectively.
```
etcd is all lowercase (comment holds throughout doc)

Reply:
done, thanks :)